spec/components/schemas/Payments/RequestSources/10_PaymentRequestPayPalSource.yaml

```
@@ -0,0 +1,11 @@
+type: object
+description: PayPal Source
+required:
+  - reference
+allOf:
+  - $ref: "#/components/schemas/PaymentRequestSource"
+  - type: object
+    properties:
+      reference:
+        type: string
+        description: A unique payment reference identifer
```

I would be more explicit and say
> A unique payment reference ~identifer~ *identifier* **per successful payment**

as the same identifier can be reused to request a payment that failed for any reason.

---

@philippe-leonhardt-cko Yeah, I should be more specific. 
But is it 'per successful payment' or just 'per payment'? I think maybe the second

---

```suggestion
        description: A unique payment reference identifier per payment
```

---

Ah okay, that makes sense then, so:
```suggestion
        description: A unique payment reference identifier per successful payment
```